.ansible-lint

```diff
@@ -17,9 +16,12 @@ kinds:
   - playbook: "**/examples/*.yml"
 skip_list:
   - fqcn-builtins
+warn_list:
+  - load-failure
```

Comment:
It seems that the ansible-lint changes here are not intentional, right?  Because these changes would make the ansible-lint broken.